_core/smart/tutorial.md

```diff
@@ -1,308 +1,208 @@
-
-Take a moment to `cat` the contents of the `sample-programs/names.clar` file.
-
-```bash
-cat sample-programs/names.clar
-````
-
-Which `tokens.clar` function is being called? 
-
-## Task 3: Initialize data-space and launch contracts
-
-In this task, you interact with the the contracts using the `clarity-cli` command line. 
-
-1. Initialize a new `db` database in the `/data/` directory
+3. The project is located in a new folder, `clarity-hello-world` by default. Jump into the folder and have a look at the file structure:
 
     ```bash
-    #  clarity-cli initialize /data/db
-    Database created
+    cd clarity-hello-world
+    ls
     ```
 
-    You should see a message saying `Database created`. The command creates an SQLlite database.  The database is available in the container and also in your workstation. In this tutorial, your workstation mount should, at this point, contain the `$HOME/blockstack-dev-data/db`  directory.
+    Take note of the `contracts` and `test` folders. The other files are boilerplate to wire up the project.
 
-2. Type check the `names.clar` contract.
+## Step 2: Reviewing hello world contract
 
-    ```bash
-    #  clarity-cli check sample-programs/names.clar /data/db
-    ```
-    
-    You should get an error:
+Now, let's have a look at a Clarity smart contract and get familiar with the basic language design characteristics.
 
-    ```
-    Error (line 11, column 1): use of unresolved contract ''S1G2081040G2081040G2081040G208105NK8PE5.tokens'.
+1. Still inside the terminal, list the contents of the `contracts` folder.
+
+    ```bash
+    ls contracts
     ```
 
-    This happens because the `names.clar` contract _calls_ the `tokens.clar` contract, and that contract has not been created on the blockchain.
+    This directory contains one file for the hello world smart contract. Note that all Clarity files have a `.clar` suffix.
 
-3. Type check the `tokens.clar` contract, it should pass a check as it does not use the `contract-call` function:
+2. Let's review the contents of `hello-world.clar` with the `cat` command.
 
     ```bash
-    # clarity-cli check sample-programs/tokens.clar /data/db
-    Checks passed.
+    cat contracts/hello-world.clar
     ```
 
-    When the `check` command executes successfully and exits with the stand UNIX `0` exit code.
-
-4. Generate a demo Stacks address for testing your contract.
+    You should see the contract source code. Take a few seconds to review the content.
 
-   This address is used to name your contract at launch time. You can use any existing Stacks address. For this sample, you are going to use the `generate_address` command to create one.
+    Clarity is a programming language based on [LISP](https://en.wikipedia.org/wiki/Lisp_(programming_language)). Most notably, Clarity is designed for static analysis, not compiled, and **not** [Turing complete](https://en.wikipedia.org/wiki/Turing_completeness).
```

Comment:
Same idea here -- we've historically been phrasing Clarity features in the affirmative, since we don't want to frame these key features as if they were limitations.